cg/apps/scoutapi.py

```diff
@@ -52,10 +52,14 @@ def get_genes(self, panel_id: str, version: str=None) -> list:
             panel genes: panel genes list
         """
         gene_panel = self.gene_panel(panel_id=panel_id, version=version)
-        return gene_panel.get('genes')
 
-    def get_cases(self, case_id=None, institute=None, reruns=None, finished=None, causatives=None, research_requested=None,
-          is_research=None, status=None):
+        if gene_panel:
```

Comment:
Is there any way we can have `self.gene_panel` raise an exception when no panels are found instead of returning None? And then we catch the exception here instead of checking if the expression evaluates to False.

Reply:
removed